bootstrap/src/main/java/org/jboss/windup/bootstrap/Bootstrap.java

```diff
@@ -458,19 +458,19 @@ private boolean executePhase(CommandPhase phase, CopyOnWriteArrayList<Command> c
         return true;
     }
 
-    private File getWindupAddonsDir()
-    {
-        return new File(getWindupHome(), "addons");
-    }
-
-    private File getWindupHome()
-    {
-        String windupHome = System.getProperty(WINDUP_HOME);
-        if (windupHome == null)
-        {
-            Path path = new File("").toPath();
-            return path.toFile();
-        }
-        return Paths.get(windupHome).toFile();
-    }
+//    private File getWindupAddonsDir()
+//    {
+//        return new File(getWindupHome(), "addons");
+//    }
+
+//    private File getWindupHome()
+//    {
+//        String windupHome = System.getProperty(WINDUP_HOME);
+//        if (windupHome == null)
+//        {
+//            Path path = new File("").toPath();
+//            return path.toFile();
+//        }
+//        return Paths.get(windupHome).toFile();
+//    }
```

Comment:
Not related, but nowhere used so commenting them

Reply:
ok